mayland-blocks/style.css

```diff
@@ -0,0 +1,87 @@
+/*
+Theme Name: Mayland (Blocks)
+Theme URI: https://github.com/Automattic/themes/tree/master/mayland-blocks
+Author: Automattic
+Author URI: https://automattic.com/
+Description: Make your online portfolio wonderfully uncluttered with Mayland. Gracefully highlight your photography and other projects. Mayland is versatile enough to be your personal site too.
+Requires at least: WordPress 4.9.6
+Version: 1.2.9
+License: GNU General Public License v2 or later
+License URI: LICENSE
+Text Domain: mayland-blocks
+Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-menu, custom-logo, editor-style, featured-images, footer-widgets, rtl-language-support, sticky-post, threaded-comments, translation-ready, auto-loading-homepage, jetpack-global-styles
+
+This theme, like WordPress, is licensed under the GPL.
+Use it to make something cool, have fun, and share what you've learned with others.
+*/
+
+
+/* Smooth out the fonts. */
+
+body {
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
```

Comment:
This feels like it should be part of Gutenberg

Reply:
Not sure if every theme will want to have this on their stylesheet, feels like something we want for this specific font family.

What is under `/* Set editor widths. */` would be something we'd be able to remove once the alignment PR lands most likely. 
The rest of the styles (header spacing and letter spacing) are dependant on https://github.com/WordPress/gutenberg/issues/23294 and https://github.com/WordPress/gutenberg/issues/23228 respectively